lint.go

```diff
@@ -1053,7 +1056,11 @@ func (f *file) lintErrorf() {
 		if !ok || !isPkgDot(ce.Fun, "fmt", "Sprintf") {
 			return true
 		}
-		f.errorf(node, 1, category("errors"), "should replace errors.New(fmt.Sprintf(...)) with fmt.Errorf(...)")
+		errorfPrefix := "fmt"
```

Comment:
ugh, I see. I didn't think this through enough. So we need the bools from above to make this switch correctly. okay, leave the previous code as it was (except for hoisting the `len(ce.Args) != 1` check, since it is so cheap).